CHANGES.rst

```diff
@@ -1,6 +1,23 @@
 Changes
 =======
 
+Version 0.5.0 (2019-04-23)
+--------------------------
+
```

Comment:
Note that release news for the REANA repository are a bit different... This is a place where people may want to find all the high-level news about the REANA platform as such. So, basically, the CHANGES here have been aggregating the main news items from REANA cluster and REANA client.

We could reformulate in this sense, but then we'd loose the nice developer-oriented news items you have compiled... Let's muse IRL...

Reply:
Can you try something like this?

```
- Users:
  - Allows to explore workflow results by running interactive Jupyter notebook sessions on the workspace files.
  - Allows to declare computing resources needed for workflow runs, such as access to CVMFS repositories.
  - Improves ``reana-client`` command-line client with new options to stop workflows, diff workflows, move and remove files.
  - Upgrades CWL engine to 1.0.20181118133959.
  - See additional changes in [reana-client release notes](https://reana-client.readthedocs.io/en/latest/changes.html#version-0-5-0-2019-04-24).
- Administrators:
  - Upgrades to Kubernetes 1.14, Helm 2.13 and Minikube 1.0.
  - Separates cluster infrastructure pods from runtime workflow engine pods that will be created by workflow controller.
  - Introduces configurable CVMFS and CephFS shared volume mounts.
  - Adds support for optional HTTPS protocol termination.
  - Introduces incoming workflow queue for additional safety in case of user storms.
  - Makes infrastructure pods container image slimmer to reduce the memory footprint.
  - See additional changes in [reana-cluster release notes](https://reana-cluster.readthedocs.io/en/latest/changes.html#version-0-5-0-2019-04-24).
- Developers:
  - Enhances development process by using git-submodule-like behaviour for shared components. 
  - Introduces simple Makefile for (fast) local testing and (slow) nightly building purposes.
  - Centralises logging level and common Celery tasks.
  - Adds helpers for test suite fixtures and improves code coverage.
```

Reply:
... alternatively, we could compile the full list split by component:

```
- REANA-Commons
  - ...
- REANA-Client
  - ...
- REANA-Job-Controller
  - ...
```